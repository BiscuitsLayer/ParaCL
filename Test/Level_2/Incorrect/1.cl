temp = func (x, y) : recursive {
    if (x == y) {
        print (-500);
    }
    else {
        recursive (x + 1, y);
    }
}

recursive (0, 100000);