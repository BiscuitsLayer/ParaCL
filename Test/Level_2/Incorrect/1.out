make: *** [Makefile:12: run] Ошибка сегментирования (сделан дамп памяти)